api_tests/dry/multiple_peers.js

```diff
@@ -1,5 +1,6 @@
 const chai = require("chai");
 const Web3 = require("web3");
+const Utils = require("web3-utils");
```

Comment:
:thinking: I think in JS, only classes are named Uppercase. Modules should be lowercase.